tools/run_local_ironic.sh

```diff
@@ -2,14 +2,34 @@
 
 set -ex
 
-SCRIPTPATH="$(dirname "$(readlink -f "${0}")")"
-
 IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
 IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metal3-io/ironic-inspector"}
 IRONIC_KEEPALIVED_IMAGE=${IRONIC_KEEPALIVED_IMAGE:-"quay.io/metal3-io/keepalived"}
 IPA_DOWNLOADER_IMAGE=${IPA_DOWNLOADER_IMAGE:-"quay.io/metal3-io/ironic-ipa-downloader:master"}
 IRONIC_DATA_DIR=${IRONIC_DATA_DIR:-"/opt/metal3-dev-env/ironic"}
 CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
+HTTP_PORT="${HTTP_PORT:-6180}"
+PROVISIONING_INTERFACE="${PROVISIONING_INTERFACE:-provisioning}"
+DHCP_RANGE="${DHCP_RANGE:-172.22.0.10,172.22.0.100}"
+DEPLOY_KERNEL_URL="${DEPLOY_KERNEL_URL:-http://172.22.0.1:6180/images/ironic-python-agent.kernel}"
+DEPLOY_RAMDISK_URL="${DEPLOY_RAMDISK_URL:-http://172.22.0.1:6180/images/ironic-python-agent.initramfs}"
+IRONIC_ENDPOINT="${IRONIC_ENDPOINT:-http://172.22.0.1:6385/v1/}"
+IRONIC_INSPECTOR_ENDPOINT="${IRONIC_INSPECTOR_ENDPOINT:-http://172.22.0.1:5050/v1/}"
+CACHEURL="${CACHEURL:-http://172.22.0.1/images}"
+IRONIC_FAST_TRACK="${IRONIC_FAST_TRACK:-false}"
+IRONIC_ENV_DATA_DIR=${IRONIC_ENV_DATA_DIR:-"/opt/metal3-dev-env/ironic-vars"}
```

Comment:
You probably also don't need double quotes around /opt/metal3-dev-env/ironic-vars.

Reply:
Reworked it, thanks.